index.js

```diff
@@ -180,30 +187,7 @@ morgan.format('tiny', ':method :url :status :res[content-length] - :response-tim
  * dev (colored)
  */
 
-morgan.format('dev', function developmentFormatLine (tokens, req, res) {
-  // get the status code if response written
-  var status = headersSent(res)
-    ? res.statusCode
-    : undefined
-
-  // get status color
-  var color = status >= 500 ? 31 // red
-    : status >= 400 ? 33 // yellow
-      : status >= 300 ? 36 // cyan
-        : status >= 200 ? 32 // green
-          : 0 // no color
-
-  // get colored function
-  var fn = developmentFormatLine[color]
-
-  if (!fn) {
-    // compile
-    fn = developmentFormatLine[color] = compile('\x1b[0m:method :url \x1b[' +
-      color + 'm:status\x1b[0m :response-time ms - :res[content-length]\x1b[0m')
-  }
-
-  return fn(tokens, req, res)
-})
+morgan.format('dev', ':method :url :status-colored :response-time ms - :res[content-length]')
```

Comment:
I think this is now entering breaking-change territory... the current 1.x `:dev` format will enture that un-ended color codes did not suddenly color the status message, but now the reset code has been removed from the start of the format.

Reply:
PR #30 introduced the current start code for context, as it's not there by accident currently.

Reply:
Ok - I'll add it back! Definitely not trying to introduce a breaking change here :)